### PR TITLE
Fixing "Logo consistent across lab share pages"

### PR DESCRIPTION
**Pull request by epeach**
Reverts code-dot-org/code-dot-org#15685

### CODE REVIEW

**Comment by epeach** [comment]
@aoby PTAL. Added conditional for mobile and fixed tests to work across all browsers.

File: apps/src/applab/applab.js [hunk 1]
```diff
@@ -419,6 +419,11 @@ Applab.init = function (config) {
     }
   }
 
+  //Mobile share pages do not show the logo
+  if (dom.isMobile() && config.share) {
+    $('#main-logo').hide();
+  }
+
```
Comment (aoby) [inline apps/src/applab/applab.js:426]: This is just applab. What about other shares on mobile?

File: dashboard/test/ui/features/sharepageLogo.feature [hunk 2]
```diff
@@ -1,5 +1,5 @@
-@no_mobile
 @as_student
+@no_mobile
```
Comment (aoby) [inline dashboard/test/ui/features/sharepageLogo.feature]: Should we also have a mobile-only test to make sure the logo is not present?

Comment (epeach) [reply dashboard/test/ui/features/sharepageLogo.feature]: I had implemented these at some point, but I couldn't figure out how to indicate that they were 'only-mobile'. I'll ask around and see if I can figure that out. Thanks!

Comment (epeach) [reply dashboard/test/ui/features/sharepageLogo.feature]: Alright, got it done - included scenarios for mobile behavior. PTAL! Thanks!

File: apps/src/applab/applab.js (hunk 1 above)
Comment (epeach) [reply apps/src/applab/applab.js:426]: Confirmed - other mobile shares already had this configured.

**Pull request merged**